src/update_agent/actor.rs

```diff
@@ -112,7 +113,10 @@ impl UpdateAgent {
         let report_steady = self.strategy.report_steady(&self.identity);
         let state_change =
             actix::fut::wrap_future::<_, Self>(report_steady).map(|is_steady, actor, _ctx| {
-                actor.state.steady(is_steady);
+                if is_steady {
+                    actor.state.steady();
+                    log::debug!("reached steady state, periodically polling for updates");
```

Comment:
Should this just be `"reached steady state"`? How updates are actually pulled will depend on the chosen strategy, right?

Reply:
In a recent cabal we decided that strategy only affects when the node **applies** update (i.e. reboots).
Thus with that logic, the check + deploy steps keep happening in background (periodically), till we are finally allowed to finalize a staged deployment.

As most of this logic does not exist yet (except for skeleton interfaces), we are still in time to make adjustments if the above sounds wrong.

Reply:
Gotcha, yup. That makes sense to me.